### PR TITLE
Adds pre-defined values for public tickets

### PR DESCRIPTION
**Pull request by arnuschky**
This commit adds three new settings that allow the operator to
pre-define the `queue`, `priority` and `due_date` fields for public
tickets. If one of these settings are present the corresponding input
field is hidden from the form.

The settings are the following:

    HELPDESK_PUBLIC_TICKET_QUEUE = 'website'
    HELPDESK_PUBLIC_TICKET_PRIORITY = 2
    HELPDESK_PUBLIC_TICKET_DUE_DATE = ''

If the due date is set to the empty string, no due date is saved in the
ticket (analogously to the form). The other settings should be
self-explanatory.

### CODE REVIEW

**Comment by arnuschky** [comment]
This addresses issue #503 and partially #289  and #579.

This patch was made against the 0.2.4 release as I was unsure which version I should use. I can rebase if need be.

File: helpdesk/views/public.py [hunk 1]
```diff
@@ -58,6 +59,19 @@ def homepage(request):
...
+        # add pre-defined data for public ticket
+        if getattr(settings, 'HELPDESK_PUBLIC_TICKET_QUEUE', None) is not None:
+            # get the requested queue; return an error if queue not found
+            try:
+                queue = Queue.objects.get(slug=settings.HELPDESK_PUBLIC_TICKET_QUEUE)
+            except Queue.DoesNotExist:
+                return HttpResponse(status=500)
+        if getattr(settings, 'HELPDESK_PUBLIC_TICKET_PRIORITY', None) is not None:
+            initial_data['priority'] = settings.HELPDESK_PUBLIC_TICKET_PRIORITY
+        if getattr(settings, 'HELPDESK_PUBLIC_TICKET_DUE_DATE', None) is not None:
```
Comment (willstott101) [inline helpdesk/views/public.py]: What if I want the default 'due_date' to be `None`?

Comment (willstott101) [reply helpdesk/views/public.py]: Right, I missed your initial comment. Would it not just be easier to use `hasattr` as opposed to having special values?

Comment (arnuschky) [reply helpdesk/views/public.py]: Err, yes, I guess so. :) I was unhappy with the special value but couldn't think of an easy way to get rid of it.

Regarding None: As we pre-set this value via a hidden in the form, I think we always end up with an empty string. I should test this though, I am not 100% sure of Django's behavior here.

Comment (arnuschky) [reply helpdesk/views/public.py]: It's actually the other way round: the empty string always gets converted to `None` for datetime fields. I updated the PR accordingly.

**Comment by gwasser** [comment]
Looks pretty good and simple. I'll wait to merge in case you're working on documentation updates. Thanks for the contribution!

**Pull request merged**